### DISCUSSION TITLE
SetValue on TAG ContourSequence throws java.lang.UnsupportedOperationException . Using dcm4che-core 5.23.1

### DISCUSSION THREAD

**Discussion by Homersbm** [dcm4che/dcm4che#925] [Q&A] [answered]
Set Value on Tag.ContourSequence seem to have a problem. As when I tried to set any form of Object (Attribute or Array of Attributes) throws **java.lang.UnsupportedOperationException** while  writeDataset.

What I noticed , ContourSequence seem to accept only Byte Array

Attributes final = new Attributes()

Sequence roiSeq =  new Sequence
Attributes temp = new Attributes();

Attributes  ss = new Attributes();
**ss.setValue(Tag.ContourSequence, VR.SQ,temp); // Problem area**

roiSeq.set(0,ss);
final.setValue(Tag.ROIContourSequence, VR.SQ, roiSeq);

File dest = new File(contourFileName);
DicomOutputStream dos = new DicomOutputStream(dest);
dos.writeDataset(null, final);

java.lang.UnsupportedOperationException.

I am currently Using dcm4che-core 5.23.1

**Comment by gunterze** [comment, marked as answer]
s. #924
